lab-gurpreet/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gulpapp",
+  "version": "1.0.0",
+  "description": "installing gulp and dependencies ",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "eslint": "^3.0.1",
+    "gulp": "^3.9.1",
```

Comment:
gulp and eslint should be devDependencies. Your program doesn't need them to run. Get rid of the debug dependency since you're not using it.